consensus/mint-client/src/bin/main.rs

```diff
@@ -60,9 +60,27 @@ fn main() {
             exit(resp.get_result().get_code().value());
         }
 
-        Commands::GenerateMintConfigTx { out, params } => {
+        Commands::GenerateMintConfigTx {
+            out,
+            tombstone_from_node,
+            params,
+        } => {
             let tx = params
-                .try_into_mint_config_tx(|| panic!("missing tombstone block"))
+                .try_into_mint_config_tx(|| {
+                    if let Some(node) = &tombstone_from_node {
+                        let env =
+                            Arc::new(EnvBuilder::new().name_prefix("mint-client-grpc").build());
```

Comment:
in master branch, this was changed so that `let env = ...` occurs once at the beginning no matter what command we invoke

https://github.com/mobilecoinfoundation/mobilecoin/blob/master/consensus/mint-client/src/bin/main.rs#L28

The review comment that prompted this was here:

https://github.com/mobilecoinfoundation/mobilecoin/pull/2584#discussion_r1009916953